server/service/vulnerabilities.go

```diff
@@ -42,10 +43,18 @@ func listVulnerabilitiesEndpoint(ctx context.Context, req interface{}, svc fleet
 		return listVulnerabilitiesResponse{Err: err}, nil
 	}
 
+	updatedAt := time.Now()
+	for _, vuln := range vulns {
+		if vuln.HostsCountUpdatedAt.Before(updatedAt) {
```

Comment:
Any performance risk here?
I'm not sure if/how much  access to DB happens here.

Reply:
the data is already returned from the database here, so there should be minimal effect as we iterate through the vulns